### PR TITLE
Closes #40 - Upgrade to Python 3.8

### LINKED ISSUE #40: Update to Python 3.8
Arkouda no only supports Python 3.8+. Any `setup.py` files referencing Python version will need to be updated to `python>=3.8`

### PR DESCRIPTION
**Pull request by Ethan-DeBandi99**
Closes #40 

- Updates `setup.py` files for clients to require `python>=3.8`
- Ran `black` and `isort` to format.
- Updated `akgraph`, `aksparse`, and `aksolve` files to properly use elements imported from `setuptools`.

### CODE REVIEW

**Review by mhmerrill — APPROVED** [review]
looks ok to me.

File: akgraph/client/setup.py [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-    setup_requires=['setuptools_scm'],
-    install_requires=['h5py','numpy'],
-    python_requires='>=3.6',
+    setup_requires=["setuptools_scm"],
+    install_requires=["h5py", "numpy"],
+    python_requires=">=3.8",
     project_urls={
-        'Bug Reports': 'https://github.com/Bears-R-Us/arkouda/issues',
-    }
+        "Bug Reports": "https://github.com/Bears-R-Us/arkouda/issues",
```
Comment (pierce314159) [inline akgraph/client/setup.py:25]: should we change this to point to `https://github.com/Bears-R-Us/arkouda-contrib/issues`?

**Review by pierce314159 — APPROVED** [review]
Question about `project_urls` but looks good otherwise!

File: akgraph/client/setup.py (hunk 1 above)
Comment (mhmerrill) [reply akgraph/client/setup.py:25]: yes

Comment (mhmerrill) [reply akgraph/client/setup.py:25]: @Ethan-DeBandi99 could you fix this so we can merge it? thanks

Comment (Ethan-DeBandi99) [reply akgraph/client/setup.py]: This has been updated now. Sorry for the delay!

**Pull request merged**